avocado/core/test.py

```diff
@@ -369,6 +369,8 @@ def record_and_warn(*args, **kwargs):
         self.__log_warn_used = False
         self.log.warn = self.log.warning = record_and_warn
 
+        self.log.info('INIT %s', self.name)
+
```

Comment:
yes, it looked a bit odd to me previously as well, but I got used to it. Anyway tagging INIT makes sense, just considering whether we should go with python `__init__`, compromise `INIT` or test-like `LOAD` (as we load the test). Other alternatives would be `IMPORT`, `INITIALIZE`, `BEGIN`.

I like `INIT` the most, but I provided those example for you consideration (even though I know you already considered probably all of them, but I might have accidentally included some you might like more).

Reply:
I thought about that too, and to me, it seems honest enough to log `INIT` at the test class `__int__()`.  And, that doesn't conflict with other stages we may be interested in logging in the future, such as `LOAD` that you mentioned.